jsonschema-core.xml

```diff
@@ -324,10 +309,13 @@
 
         <section title="Schema references with $ref">
             <t>
-                In addition to a boolean value or an object using schema kewyords defined
-                in the meta-schema, a schema may be represnted by an object containing a "$ref" property.
-                The value of the $ref is a URI Reference.
+                The "$ref" keyword is used to reference an externally defined schema, and provides the ability to validate recursive structures through self-reference.
+            </t>
+            <t>
+                An object subschema with a "$ref" property MUST be interpreted as a "$ref" reference.
+                The value of the "$ref" property MUST be a URI Reference.
                 Resolved against the current URI base, it identifies the URI of a schema to use.
+                The referenced schema MUST NOT itself contain a "$ref" property.
```

Comment:
The issue the $ref behavior is only guaranteed for subschemas, this is one way of enforcing that.

The behavior can be done by placing it inside an "allOf" if desired.

But I suppose this is an issue for a... new issue anyways.